### DISCUSSION TITLE
If you are in search of Starstruck Achivement 👇

### DISCUSSION THREAD

**Discussion by lokesh7suman** [Schweinepriester/github-profile-achievements#277] [General]
Hello folks,

Star My repo. [https://github.com/d4rkvent/fonts.git](url)
I will do the same.
leave the link to your repo which you want to be stared..

**Comment by PaulSpoerry** [comment]
Done. Mine: https://github.com/PaulSpoerry/test

FYI: You need to remove the .git from your URL above.
